.github/workflows/build.yml

```diff
@@ -69,6 +69,8 @@ jobs:
       image: swift:${{ matrix.swift }}${{ matrix.ubuntu }}
     steps:
     - name: Checkout
+      # 5.1.5 and lower Swift versions end up on images where actions/checkout@v4
+      # fails because the image is too old.
```

Comment:
I don't know if at some point github will be making these start to fail, so that might force the branch to drop some swift version coverage.

Reply:
I guess we can wait until/if that happens. I wonder how many people would miss coverage of versions that old.